skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -874,5 +874,45 @@ def test_from_dict_to_dict_roundtrip(self):
         self.assertIs(d1[42], d2[42])
 
 
+class TestIterPositions(unittest.TestCase):
+    def setUp(self):
+        self.msa_empty = TabularMSA(())
+        self.msa = TabularMSA((DNA('ACGT'), DNA('TGCA')))
+        self.expected = [[Sequence('A'), Sequence('T')],
```

Comment:
Also we have `.frequencies()` on `Sequence` now. So that gives us positional frequencies for free.